tasks/RedHat.yml

```diff
@@ -12,7 +12,7 @@
 - name: Download package locally
   become: no
   connection: local
-  get_url: "url={{ couchbase_server_centos_ee_url }} dest={{ role_path }}/files/{{ couchbase_server_centos_ee_package }} sha256sum={{ couchbase_server_centos_ee_sha256 }} timeout=240"
+  get_url: "url={{ couchbase_server_centos_ee_url }} dest={{ role_path }}/files/{{ couchbase_server_centos_ee_package }} mode=0644 sha256sum={{ couchbase_server_centos_ee_sha256 }} timeout=240"
```

Comment:
Because, after the download, we had an rpm file with mode root:root 600  at the end of the download and the next task (copy) failed. This seems to be the default also unzipping files on the RedHat machine or maybe is a default behaviour that could be setup (umask). Then, It's better to explicitly require needed permissions.